plugins/batterystat.koplugin/main.lua

```diff
@@ -207,10 +203,47 @@ function BatteryStat:showStatistics()
     table.insert(kv_pairs, "----------")
     table.insert(kv_pairs, {_("Historical records are dumped to"), ""})
     table.insert(kv_pairs, {self.dump_file, ""})
-    UIManager:show(KeyValuePage:new{
+    table.insert(kv_pairs, "----------")
+    table.insert(kv_pairs, {_("Should you like to reset the data,"), "",
```

Comment:
*Should like* is a rather formal way of saying *would like* and possibly mostly British. I'd stick with "If you want to reset the data" or "If you would like to reset the data".